composer.json

```diff
@@ -28,7 +28,7 @@
     "require-dev": {
         "phpspec/prophecy": "^1.8",
         "psr/container": "^1.0",
-        "symfony/http-foundation": "^3.4 || ^4.2|| ^5.0",
+        "symfony/http-foundation": "^5.1",
```

Comment:
We can't exclude versions 3 and 4 of Symfony

Reply:
oh, I see ! I will modify this, thank !